code/02-nix-commands/02-use-packages-in-nix-shell.md

```diff
@@ -23,4 +22,25 @@ $ nix-shell -p nodejs ghc cabal-install
 
 [nix-shell:nix-workshop]$ which cabal
 /nix/store/060x141b9fz2pm6yz4zn3i0ncavbdbf7-cabal-install-3.2.0.0/bin/cabal
-```
\ No newline at end of file
+```
+
+## Using `shell.nix`
+
+`nix-shell` will read `shell.nix` if it exists. We go into details later; for now all you need to know is that if you see a `shell.nix` in a project, you can run `nix-shell` to get a temporary environment. This is one of Nix's killer features: it gives you the power of a tool like `venv`, but for any program build in any language.
```

Comment:
This section may be too confusing to the readers. We have not introduced the Nix syntax, so readers would just get lost on what the example `shell.nix` does.

Not every one knows about power tools like `venv`. So that is not enough to motivate readers to use Nix shells.

We can just show a link, mentioning that in later chapter we will cover how to create a `shell.nix` file to setup a development environment.

Reply:
Reworded.